extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/devconsole/QuteDevConsoleProcessor.java

```diff
@@ -122,7 +118,7 @@ private CheckedTemplateBuildItem findCheckedTemplate(String basePath, List<Check
         for (String variant : variants) {
             String source = templatePaths.stream().filter(p -> p.getPath().equals(variant))
                     .map(TemplatePathBuildItem::getContent).findFirst()
-                    .orElse(null);
+                    .orElse("");
```

Comment:
I'm not sure it's the right fix. Do we want an entry if no source? Let's ask @mkouba !

Reply:
It was the previous behavior, but yeah, let's wait on @mkouba :)

Reply:
I think it's OK, although it should never happen...